main/settings.py

```diff
@@ -37,8 +37,9 @@
     'django.contrib.sessions',
     'django.contrib.messages',
     'django.contrib.staticfiles',
-    'avtosalon.apps.AvtosalonConfig',
+    'showroom.apps.AvtosalonConfig',
```

Comment:
differ local apps and django defualt apps. add `#django_apps #local_apps #3rd-party_apps` comments to differ them